src/views/kbv/single-amount-question.njk

```diff
@@ -4,7 +4,7 @@
 
 {% set gtmJourney = "kbv-hmrc - middle" %}
 
-{% set hmpoTitle = question.key %}
+{% set hmpoTitle = translate("pages.rti-payslip-national-insurance.title") %}
 
 {% block mainContent %}
   <h1 class="govuk-heading-l"> {{ translate("pages.rti-payslip-national-insurance.title") }} </h1>
```

Comment:
As this template will be shared across multiple questions, we'll need a way of passing in the question key for the translations, maybe using `presenters` to build the content instead